package.json

```diff
@@ -47,6 +47,7 @@
     "lodash.merge": "^4.3.5",
     "lodash.throttle": "^4.0.1",
     "react-addons-create-fragment": "^15.0.1",
+    "react-addons-shallow-compare": "^15.0.1",
```

Comment:
@oliviertassinari We are currently using it from `recompose`: https://github.com/callemall/material-ui/blob/master/src/TextField/TextField.js#L4

But I think we should get rid of `recompose`.... all we are using it for is `pure()` which is an entire component instance just to wrap a component with:

```js
class Pure extends Component {
  
  shouldComponentUpdate(nextProps) {
    return !shallowEqual(this.props, nextProps);
  }

  render() {}
}
```